policyengine_us/parameters/gov/states/la/tax/credits/child_care_expense_credit/rate/la_state_credit.yaml

```diff
@@ -0,0 +1,25 @@
+description: Louisiana provides this percentage of federal income tax credit for state child care credit to which the individual resident's federal agi is less than sixty-thousand dollars
```

Comment:
Same thing here with the description. Please consider create another parameter file for the $6,000.